internal/provider/resources/resource_api_token_test.go

```diff
@@ -371,27 +351,7 @@ func TestAcc_ResourceWorkspaceApiToken(t *testing.T) {
 						},
 					},
 				}),
-				ExpectError: regexp.MustCompile("Role 'ORGANIZATION_OWNER' is not valid for token type 'WORKSPACE'"),
-			},
-			// Test multiple roles of the same type
```

Comment:
this is the faulty test

Reply:
since creation doesn’t let you do multiple so that passes but updating roles does so the create passes and update fails so what is left is the dangling token we created in the create request